modules/aws/s3_test.go

```diff
@@ -169,6 +170,17 @@ func testEmptyBucket(t *testing.T, s3Client *s3.S3, region string, s3BucketName
 			t.Fatal(err)
 		}
 
+		if i < 10 {
```

Comment:
Could you add a comment explaining what you're doing here? It's obvious in context of this PR, but won't be for a maintainer a few months from now.

Reply:
Sure, done